app/chitterapp.rb

```diff
@@ -0,0 +1,31 @@
+ENV['RACK_ENV'] ||= 'development'
+
+require 'sinatra/base'
+require 'sinatra/flash'
+require 'sinatra/partial'
+
+require_relative 'models/init'
+require_relative 'controllers/init'
+require_relative 'helpers'
+
+class ChitterApp < Sinatra::Base
+  register Sinatra::Flash
+  register Sinatra::Partial
+
+  use Rack::MethodOverride
+
+  set :views, Proc.new { File.join(root, "../views") }
```

Comment:
Style/Proc: Use `proc` instead of `Proc.new`. (https://github.com/bbatsov/ruby-style-guide#proc)